tests/visco_plastic_peierls_mei_viscosity_approx.prm

```diff
@@ -0,0 +1,118 @@
+# A test for the peierls creep viscositiy approximation  
+# with the visco_plastic material model. 
+# Following the general Peierls creep flow law of Mei 
+# et al. 2010 and assuming a stress of 900e6 Pa, the
+# exact viscosity is calculated in python through:
+#  import numpy as np
+#  R    = 8.314  # gas constant [J/mol*K] 
+#  g    = 9.81   # gravity [m/s^2]
+#  T    = 1073   # temperature [K]
+#  sig  = 900e6  # stress [Pa] 
+#  sigp = 5.9e9  # Peierls creep stress [Pa]
+#  Ap   = 1.4e-7 /np.power(1e6,2) # Peierls creep prefactor 1/s 1/Pa^2 
+#  E    = 320e3  # Peierls creeep activation energy [J/mol]
+# 
+#  Strain rate invariant:
+#    edot_ii = Ap*np.power(sig,2)*np.exp( -(E/(R*T))*(1 - np.sqrt(sig/sigp)) ) = 3.6356390302747323e-11
+#
+#  Viscosity:
+#    visc = 0.5*sig/edot_ii = 1.237746641657093e+19
+#
+#  Fitting parameters for approximated viscosity:
+#    q = 0.5
+#    g = 0.1
```

Comment:
But g is also gravitational acceleration (see above)? Maybe a different parameter name would avoid confusion?

Reply:
will switch `g` to `gamma`